src/thing_creature.c

```diff
@@ -4527,24 +4536,25 @@ struct Thing *create_owned_special_digger(MapCoord x, MapCoord y, PlayerNumber o
     pos.x.val = x;
     pos.y.val = y;
     pos.z.val = 0;
-    struct Thing* thing = create_creature(&pos, crmodel, owner);
+    struct Thing *thing = create_creature(&pos, crmodel, owner);
     if (thing_is_invalid(thing))
     {
-        ERRORLOG("Cannot create creature %s at (%ld,%ld)",creature_code_name(crmodel),x,y);
+        ERRORLOG("Cannot create creature %s at (%ld,%ld)", creature_code_name(crmodel), x, y);
         return INVALID_THING;
     }
     pos.z.val = get_thing_height_at(thing, &pos);
     if (thing_in_wall_at(thing, &pos))
     {
         delete_thing_structure(thing, 0);
-        ERRORLOG("Creature %s at (%ld,%ld) deleted because is in wall",creature_code_name(crmodel),x,y);
+        ERRORLOG("Creature %s at (%ld,%ld) deleted because is in wall", creature_code_name(crmodel), x, y);
         return INVALID_THING;
     }
     thing->mappos.x.val = pos.x.val;
     thing->mappos.y.val = pos.y.val;
     thing->mappos.z.val = pos.z.val;
     remove_first_creature(thing);
     set_first_creature(thing);
+    set_start_state(thing);
```

Comment:
You say just a formatting change, but this is a functional change

Reply:
if it does nothing, then please do not make the change. We introduce enough bugs on code we change for a reason, to start changing code that we thing would make no change.